web/themes/custom/vote_gov/templates/block/block-content--registration-tool.html.twig

```diff
@@ -0,0 +1,99 @@
+{% block content %}
+<div class="bg-blue css-arrow page-vote">
+    <div class="grid-container">
+      <div class="grid-row grid-gap">
+        <div class="tablet:grid-col-4">
+		  {% set logo_sitename = language != 'en' ? sitename ~ ' in ' ~ languagename : sitename %}
+          <a class="site-logo" href="/" aria-label="{{ logo_sitename | t }}">
+            <div class="logo-text">{{ logo_sitename }}</div>
+            <div id="SiteLogo">{{ source( directory ~ '/img/ballot_box-blue.svg' ) }}</div>
+			    </a>
+        </div>
+            <div id="skip-target" class="tablet:grid-col-8 right-shift">
+              <section class="register align-bottom">
+                <header>
+                  <h1 class="main-heading" data-test="main-header">{{ content.field_heading | field_value }}</h1>
+                </header>
+
+                <form class="form-register" method="get" id="register">
+                  <h2>{{ content.field_form_heading | field_value }}</h2>
+                  <label class="usa-sr-only" for="js-user-selection">
+                    {{ content.field_dropdown_label | field_value }}
+                  </label>
+                  <select class="usa-select max-width-input" name="userselection" id="js-user-selection" data-test="state-selection">
+                    <option value="default">
+                      {{ content.field_dropdown_label | field_value }}
+                    </option>
+                    {% set states = [
```

Comment:
Please make a ticket in the backlog to convert this to dynamically pull all state terrirory pages.
You can add this link as a resource on how we might do it in a preprocess
https://drupal.stackexchange.com/questions/213689/get-all-nodes-of-given-type